### DISCUSSION TITLE
Create new workout

### DISCUSSION THREAD

**Discussion by beradrian** [zepp-health/zeppos-forum#12] [Developer Feedback]
Is it possible to create a new type of workout based on an existing one, that only has a different screen?

I'm thinking that if it will exist something like startWorkout('running") and then you can customize the screen it will be helpful.

Thanks!

**Comment by sirAvent** [comment]
I believe on the watch's workouts page; you can add and edit pages for existing workouts.

Once you go on a workout page, let's say "Walking", you can scroll down to "Settings", press on "More", and then "Data Page" to start making a custom workout screen.

So technically you're not making a new workout, but you're making a new screen of an existing workout to view the data you want. I hope this answers your question!

**Reply by beradrian** [reply]
> So technically you're not making a new workout, but you're making a new screen of an existing workout to view the data you want. I hope this answers your question!

I know this, but the thing is that I want to have the workout display (and the user to have access) ONLY some data, not all. And also to be able to create a completely new screen. This will allow developers to create new custom workouts, not only apps.

Or looking from another perspective, if the API will expose a method, as I said earlier, to start tracking a new workout with some predefined options (and being able to interact with that, e.g. when a new record is registered - e.g. new GPS position) then we will be able to achieve the same thing.

**Comment by MyoungXUE** [comment]
Thanks for your support to the forum.

For now we don't support this feature, but I think it's a very cool idea and I'll feed it to the relevant colleagues.
